champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/sword/StaticLazer.java

```diff
@@ -141,6 +140,7 @@ public void loadSkillConfig(){
         energyPerSecond = getConfig("energyPerSecond", 24.0, Double.class);
         collisionRadius = getConfig("collisionRadius", 1.8, Double.class);
         explosionRadius = getConfig("explosionRadius", 4.0, Double.class);
+        energyPerSecondReductionPerLevel = getConfig("energyPerSecondReductionPerLevel", 2.0, Double.class);
```

Comment:
I think adding energyPerSecond and energyPerSecondReductionPerLevel were a mistake here (probably reys fault)

All energy skills have a default configuration

![image](https://github.com/Mykindos/BetterPvP/assets/5389944/21590839-1695-4f4b-96dd-5c4943dc1fe4)

Can you change it to use these instead

Reply:
energyPerSecondReductionPerLevel is for the energy consumed per second, the skill also has an initial energy cost which is defined by energy and energyDecreasePerLevel

Reply:
No other skill does that, we typically just use the base energy cost as the activation cost, or pick a relatively low number

Reply:
well both static and now this do. I can change them both to use a higher amount and drain over time